presto-orc/src/main/java/com/facebook/presto/orc/metadata/statistics/StringStatistics.java

```diff
@@ -98,4 +99,12 @@ public String toString()
                 .add("sum", sum)
                 .toString();
     }
+
+    @Override
+    public void addHash(StatisticsHasher hasher)
```

Comment:
The `MetadataReader` will make `StringStatistics` null if both `min` and `max` are null. So if we write down a string stats with only `sum != null`, the hash will be different.